### PR TITLE
Update testimonials section of the homepage

### LINKED ISSUE #72: Update testimonials section of the homepage
This is to update the testimonial section of the homepage with actual testimonials provided (Content has been shared with you)

### PR DESCRIPTION
**Pull request by OdunayoOkebunmi**
- remove dummy testimonials 
- update it with real testimonials
Fixes #72

### CODE REVIEW

File: templates/views/index.html [hunk 1]
```diff
@@ -80,30 +85,40 @@ <h4>Support our cause</h4>
...
+                <img src="/images/PeaceAmara.jpg" class="__shecodetestimimg">
             </div>
             <div class="col-lg-4 col-md-4 col-sm-4 text-center">
-                <div class="__shecodetestimcard">
-                    <p class="__shecodetestimp">Ea nam iudico equidem meliore. Ea eam dico alia fabulas. Eu vulputate forensibus scribentur vel, mei in officiis eloquentiam. Esse platonem sit no, amet velit nemore in quo. Ad eos rationibus reprehendunt, ea tollit dictas volutpat
-                        eos. Et iuvaret probatus voluptatum mea, vix an urbanitas constituto, ex labores accusata est.
+                <div class="__shecodetestimcard last-child">
+                    <p class="__shecodetestimp">I met lots of people that has been helping in my journey so far,also
+                        privilledge to be part of the COHORT 1 and I've
+                        learnt so much within a short period.
```
Comment (mazma1) [inline templates/views/index.html]: This testimonial needs some correction:

I met lots of people that **have** been **helpful** in my journey so far. also **privileged** to be part of the COHORT 1 **of SCA's mentorship program** and I've learnt so much within a short period.

File: templates/views/index.html [hunk 2]
```diff
@@ -80,30 +85,40 @@ <h4>Support our cause</h4>
         <div class="row">
             <div class="col-lg-4 col-md-4 col-sm-4 text-center">
                 <div class="__shecodetestimcard">
-                    <p class="__shecodetestimp">Ea nam iudico equidem meliore. Ea eam dico alia fabulas. Eu vulputate forensibus scribentur vel, mei in officiis eloquentiam. Esse platonem sit no, amet velit nemore in quo. Ad eos rationibus reprehendunt, ea tollit dictas volutpat
-                        eos. Et iuvaret probatus voluptatum mea, vix an urbanitas constituto, ex labores accusata est.
+                    <p class="__shecodetestimp">This Sca mentoring program was like a breathe of fresh air that I needed
```
Comment (yemisi-o) [inline templates/views/index.html]: Change Sca to SCA

File: templates/views/index.html [hunk 3]
```diff
@@ -80,30 +85,40 @@ <h4>Support our cause</h4>
...
+                        since December 2019. I'm so filled as I joined
+                        and has been growing on my learning path (Php/laravel).
+                        In fact my mentor is the best that anyone can have. She's not only good but naturally a guide.
+                        God bless you @FaithAdekogbe.
+                        Thanks shecodeafrica & all the organizers. I'll forever make you guys proud. 🥰
```
Comment (yemisi-o) [inline templates/views/index.html]: Change shecodeafrica to She Code Africa

File: templates/views/index.html [hunk 4]
```diff
@@ -80,30 +85,40 @@ <h4>Support our cause</h4>
         <div class="row">
             <div class="col-lg-4 col-md-4 col-sm-4 text-center">
                 <div class="__shecodetestimcard">
-                    <p class="__shecodetestimp">Ea nam iudico equidem meliore. Ea eam dico alia fabulas. Eu vulputate forensibus scribentur vel, mei in officiis eloquentiam. Esse platonem sit no, amet velit nemore in quo. Ad eos rationibus reprehendunt, ea tollit dictas volutpat
-                        eos. Et iuvaret probatus voluptatum mea, vix an urbanitas constituto, ex labores accusata est.
+                    <p class="__shecodetestimp">This Sca mentoring program was like a breathe of fresh air that I needed
+                        since December 2019. I'm so filled as I joined
+                        and has been growing on my learning path (Php/laravel).
```
Comment (yemisi-o) [inline templates/views/index.html]: Change 'and has been growing on my learning path (Php/laravel)' to 'and I have been growing on my learning path (PHP/Laravel).'

File: templates/views/index.html [hunk 5]
```diff
@@ -80,30 +85,40 @@ <h4>Support our cause</h4>
...
                 </div>
-                <img src="/images/hadijat.jpg" class="__shecodetestimimg">
+                <img src="/images/PeaceAmara.jpg" class="__shecodetestimimg">
             </div>
             <div class="col-lg-4 col-md-4 col-sm-4 text-center">
-                <div class="__shecodetestimcard">
-                    <p class="__shecodetestimp">Ea nam iudico equidem meliore. Ea eam dico alia fabulas. Eu vulputate forensibus scribentur vel, mei in officiis eloquentiam. Esse platonem sit no, amet velit nemore in quo. Ad eos rationibus reprehendunt, ea tollit dictas volutpat
-                        eos. Et iuvaret probatus voluptatum mea, vix an urbanitas constituto, ex labores accusata est.
+                <div class="__shecodetestimcard last-child">
+                    <p class="__shecodetestimp">I met lots of people that has been helping in my journey so far,also
```
Comment (yemisi-o) [inline templates/views/index.html]: Update this testimonial to:
'Through She Code Africa, I have met lots of people that have been helping in my journey so far. I am also privileged to be part of SCA's Mentoring Program Cohort 1 and I have learnt so much within a short period.'

Also, do note that this testimonial card is still smaller/shorter than the others. They need to be equal, please adjust appropriately after updating the content

File: templates/views/index.html (hunk 1 above)
Comment (OdunayoOkebunmi) [reply templates/views/index.html]: Okay, noted

File: templates/views/index.html (hunk 5 above)
Comment (OdunayoOkebunmi) [reply templates/views/index.html]: Okay, noted

File: public/styles/main.css [hunk 6]
```diff
@@ -399,11 +399,10 @@ div.__shecodecard>div>div>div>p.__shecodevmvp1 {
     padding: 40px;
     border-radius: 5px;
     margin-top: 27%;
-
-
+    height: 27rem;
```
Comment (yemisi-o) [inline public/styles/main.css]: @OdunayoOkebunmi Is there something this height is helping us achieve with the equal heights? From the latest changes, it doesn't seem like it is needed anymore, it only adds more space below on desktop view. See below:

Without the height applied:
screenshot-sca-website-oo-72-updat-hppr4p herokuapp com-2020 03 08-17_07_07

With the height applied:
screenshot-sca-website-oo-72-updat-hppr4p herokuapp com-2020 03 08-17_08_02

Please check to see if it's still necessary

File: templates/views/index.html [hunk 7]
```diff
@@ -120,6 +110,18 @@ <h4>Support our cause</h4>
...
                 <img src="/images/PreciousCadeton.jpg" class="__shecodetestimimg">
             </div>
+            <div class="col-lg-4 col-md-4 col-sm-4 text-center">
+                <div class="__shecodetestimcard last-child">
+                    <p class="__shecodetestimp">
+                        Through She Code Africa, I have met lots of people that have been helpful in my journey so far.
+                        I am also privileged to
+                        be part of the Cohort 1 of SCA's Mentorship Program and I have learnt so much within a short
+                        period.
+                    </p>
```
Comment (yemisi-o) [inline templates/views/index.html]: Please add a line break/break tag before the name so the name is on equal length with the 2 other cards

**Pull request merged**